src/physics/initial_conditions/initial_condition.h

```diff
@@ -32,38 +48,22 @@ class FreeStreamInitialConditions : public dealii::Function<dim>
         for (int d=0;d<dim;d++) { primitive_boundary_values[1+d] = euler_physics.velocities_inf[d]; }
         primitive_boundary_values[nstate-1] = pressure_bc;
         farfield_conservative = euler_physics.convert_primitive_to_conservative(primitive_boundary_values);
+        std::cout << "Farfield conditions: "<< std::endl;
+        for (int s=0;s<nstate;s++) {
+            std::cout << farfield_conservative[s] << std::endl;
+        }
```

Comment:
When running with mpi, does this output np times? If so, we'd want to avoid that and consider printing it directly in the FlowSolverCase derived class which has knowledge of `mpi_rank`

Reply:
It does output np times. Unfortunately, the FlowSolverCase derived classes have no knowledge of the initial conditions. 

How important do you think it is to output that information? If it's important enough, I'll just add the mpi communicator to the InitialConditionFunction class and use pcout instead.

Reply:
I think what would be a good solution would be to pass `initial_condition_function` as an argument to `flow_solver_case->display_flow_solver_setup();` in the constructor of `flow_solver.cpp`, then loop over the `nstate` in your definition of `display_flow_solver_setup` for the naca0012 case to display it. This will be useful for future cases we add as well and sounds like the most appropriate place to be printing this stuff. What do you think? Just make it an unused argument in the other FlowSolverCaseBase derived classes

Reply:
Sure, done is most recent commit.